solarwinds_apm/apm_config.py

```diff
@@ -169,6 +170,9 @@ def __init__(
         self.metric_format = self._calculate_metric_format()
         self.certificates = self._calculate_certificates()
         self.__config["export_logs_enabled"] = self._calculate_logs_enabled()
+        self.__config["export_metrics_enabled"] = (
```

Comment:
Do we need the bracket? I don't see the same in line 172.

Reply:
The linter did this, probably for slightly longer line char length.